beeline/trace.py

```diff
@@ -333,7 +333,7 @@ def _run_hooks_and_send(self, span):
 
 class SynchronousTracer(Tracer):
     def __init__(self, client):
-        super(SynchronousTracer, self).__init__(client)
+        Tracer.__init__(self, client)  # noqa
```

Comment:
It'd be good to leave ourselves a reminder to Super Properly in the (hopefully near) future.

```suggestion
        Tracer.__init__(self, client)  # noqa TODO: use a Python 3.0 style super when 2.7 support is dropped
```

Reply:
lgtm!